docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,30 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed shape tensor
+==================
+
+* Extension name: `arrow.fixed_shape_tensor`.
+
+* The storage type of the extension: ``FixedSizeList`` where:
```

Comment:
Note that instead of `strides` we could store the dimensions permutation (as it seems to be the current direction) - and it could be a different one per extension array.

Reply:
I believe this was addressed and we can probably mark it as resolved?

Reply:
Yes, I think in general this was discussed, just not here but in one of the many other threads.